articles/quickstart/native/android/03-session-handling.md

```diff
@@ -199,8 +199,8 @@ private void logout() {
 Depending on the way you store users' credentials, you delete them differently. 
 :::
 
-### Optional: Encapsulate Session Handling
+## Optional: Encapsulate Session Handling
 
-Handling users' sessions is not a straightforward process. You can simplify it by storing token-related information and processes in a class. The class separates the logic for handling users' sessions from the activity. 
+Handling user sessions is not a straightforward task. You can simplify it by storing token-related information and processes in a class. The class separates the logic for handling user sessions from the activity. 
```

Comment:
In general I don't like implying difficulty e.g. `not a straightfoward` as this is subjective.